src/deserialize/ops.jl

```diff
@@ -125,6 +125,19 @@ default_Wb_Rb(Wh_WBh) = fill!(similar(Wh_WBh, (size(Wh_WBh, 2) * 2, size(Wh_WBh,
 default_init_h(Wb_Rb, sc) = fill!(similar(Wb_Rb, (size(Wb_Rb,1) ÷ sc, size(Wb_Rb,2))), 0)
 # TODO when https://github.com/FluxML/Flux.jl/issues/1279 is resolved default_init_h(Wh_WBh, sc) = fill!(similar(Wh_WBh, (size(Wh_WBh, 2) ÷ sc, size(Wh_WBh, 3))), 0)
 
+actlayers[:InstanceNormalization] = function(params, γ, β)
+    λ = get(params, :activation, identity)
+    ϵ = get(params, :epsilon, 1f-5)
+
+    # ONNX InstanceNormalization does not support tracking μ and σ²
+    momentum = NaN32
+    μ = zeros(length(γ))
+    σ² = ones(length(γ))
+
+    return InstanceNorm(λ, β, γ, μ, σ², ϵ, momentum, true, false, nothing, length(γ))
```

Comment:
I think Flux maintainers consider default constructors to be non-public API so there is less chance of breakage if one uses the documented constructor (the one where more of the above is supplied as keyword arguments). Would it be possible to use it instead?

Reply:
I tried to change it just now, but the [documented constructor](https://github.com/FluxML/Flux.jl/blob/24b1eb2f0c7de93e36d47833ae47a1b473f31f80/src/layers/normalise.jl#L418-L426) expects `initβ` and `initγ` as a function instead of an array.

I have to set β and γ directly as vectors via the struct, I suppose the same happened for `BatchNorm`.

Reply:
Ah, I had forgotten about that. Lets go with this for now then. Seems like `BatchNorm` has worked as is for along enough time to consider it stable :)